python/lsst/ip/isr/ampOffset.py

```diff
@@ -518,19 +519,20 @@ def getInterfaceOffset(self, ampIdA, ampIdB, edgeA, edgeB):
         minFracFail = ampEdgeGoodFrac < self.config.ampEdgeMinFrac
         maxOffsetFail = np.abs(interfaceOffset) > self.config.ampEdgeMaxOffset
         if minFracFail or maxOffsetFail:
```

Comment:
Hmm, maybe we should also move `if minFracFail or maxOffsetFail` inside the `doApplyAmpOffset` check?